search-handlers/src/main/java/no/unit/nva/indexing/handlers/IndexResourceHandler.java

```diff
@@ -1,24 +1,26 @@
 package no.unit.nva.indexing.handlers;
 
+import static nva.commons.core.attempt.Try.attempt;
 import com.amazonaws.services.lambda.runtime.Context;
 import no.unit.nva.events.handlers.DestinationsEventBridgeEventHandler;
 import no.unit.nva.events.models.AwsEventBridgeDetail;
 import no.unit.nva.events.models.AwsEventBridgeEvent;
+import no.unit.nva.events.models.EventReference;
 import no.unit.nva.s3.S3Driver;
 import no.unit.nva.search.IndexingClient;
 import no.unit.nva.search.IndexingConfig;
-import no.unit.nva.search.exception.SearchException;
-import no.unit.nva.search.models.IndexEvent;
 import no.unit.nva.search.models.IndexDocument;
 import nva.commons.core.JacocoGenerated;
 import nva.commons.core.paths.UnixPath;
 import nva.commons.core.paths.UriWrapper;
 
-public class IndexResourceHandler extends DestinationsEventBridgeEventHandler<IndexEvent, Void> {
+public class IndexResourceHandler extends DestinationsEventBridgeEventHandler<EventReference, Void> {
```

Comment:
Dropped IndexEvent in favor for the EventReference from nva-commons. They do the same thing only EventReference is the standard